debian/README.Debian

```diff
@@ -0,0 +1,188 @@
+Ubuntu packaging overview
+=========================
+
+Goals:
+* Working package on Ubuntu
+* Shouldn't depend on other non-main/ppa packages
+
+Ubuntu packaging overview
+-------------------------
+
+There is a lot written about Debian/Ubuntu packaging; here's a little
+more that would have helped me when I first started working on this.
+
+It helps to know that the Debian packaging tools are quite old, so
+much of the process of creating and maintaining a package is largely
+duplicating the information in a modern source code repository such as
+_git_, for lack of a better example.
+
+To upload source code to launchpad.net to be built, you need the
+following files.
+
+### an original source archive
+
+This is meant to be the original source archive that comes from the
+author of the code. It should be named something like this: `mcrouter_0.22.0.orig.tar.gz`
+
+### debian package info
```

Comment:
IMO, the `README.Debian` file shouldn't be explaining how Debian packaging works; that's not in the scope of each individual package's documentation.